docs/pages/getting_started.md

```diff
@@ -20,67 +32,108 @@ underlying data may be one of various [basic types](\ref af::af_dtype):
+determine various other aspects of arrays.
+
 The [af_print](\ref af::af_print) function can be used to print arrays that
-have already been generated or an expression involving arrays:
+have already been generated or any expression involving arrays:
 
 \snippet test/getting_started.cpp ex_getting_started_print
 
-ArrayFire provides several convenient methods for accessing the dimensions.
-You may use either a [dim4](\ref af::dim4) object or access the dimensions
-directly using the [dims()](\ref af::array::dims) and
-[numdims()](\ref af::array::numdims) functions:
+The dimensions of an array may be determined using either a
+[dim4](\ref af::dim4) object or by accessing the dimensions directly using the
+[dims()](\ref af::array::dims) and [numdims()](\ref af::array::numdims)
+functions:
 
 \snippet test/getting_started.cpp ex_getting_started_dims
 
-Arrays also provide functions to determine their properties including:
+In addition to dimensions, arrays also carry several properties including
+methods to determine the underlying type and size (in bytes). You can even
+determine whether the array is empty, real/complex, a row/column, or a scalar
+or a vector:
 
 \snippet test/getting_started.cpp ex_getting_started_prop
 
+For further information on these capabilities, we suggest you consult the
+full documentation on the [array](\ref af::array).
+
 # Writing mathematical expressions in ArrayFire {#getting_started_writing_math}
 
-Most of ArrayFire's functions operate on an element-wise basis.
-This means that function like `c[i] = a[i] + b[i]` could simply be written
-as `c = a + b`.
-ArrayFire has an intelligent runtime JIT compliation engine which converts
-array expressions into the smallest number of OpenCL/CUDA kernels.
-This "kernel fusion" technology not only decreases the number of kernel calls,
-but, more importantly, avoids extraneous global memory operations.
+ArrayFire features an intelligent Just-In-Time (JIT) compilation engine that
+converts expressions using arrays into the smallest number of CUDA/OpenCL
+kernels. For most operations on arrays, ArrayFire functions like a vector library.
+That means that an element-wise operation, like `c[i] = a[i] + b[i]` in C,
+would be written more concisely without indexing, like `c = a + b`.
+When there are multiple expressions involving arrays, ArrayFire's JIT engine
+will merge them together. This "kernel fusion" technology not only decreases
+the number of kernel calls, but, more importantly, avoids extraneous global
+memory operations.
 Our JIT functionality extends across C/C++ function boundaries and only ends
 when a non-JIT function is encountered or a synchronization operation is
 explicitly called by the code.
 
-ArrayFire has [hundreds of functions](\ref arith_mat) for element-wise
-arithmetic. Here are a few examples:
+ArrayFire provides [hundreds of functions](\ref arith_mat) for element-wise
+operations. All of the standard operators (e.g. +,-,*,/) are supported
```

Comment:
Just out of curiosity, the * here, would make all the following test italics (I see it on github. But i think it will happen in doxygen too). Can you check this. Might need to use the escape character